doc/source/index.rst

```diff
@@ -1,145 +1,63 @@
-..
-      Copyright European Organization for Nuclear Research (CERN)
-
-      Licensed under the Apache License, Version 2.0 (the "License");
-      You may not use this file except in compliance with the License.
-      You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-
-.. ATTENTION::
-    **ATLAS users should refer to** `https://twiki.cern.ch/twiki/bin/viewauth/AtlasComputing/RucioClientsHowTo <https://twiki.cern.ch/twiki/bin/viewauth/AtlasComputing/RucioClientsHowTo>`_ **instead of this page. This twiki is the entry point for ATLAS users learning the Rucio client. The ATLAS-specific information on this page will be removed.**
+.. Rucio documentation master file, created by
+   sphinx-quickstart on Fri Oct 27 14:25:40 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
 
 Welcome to Rucio's documentation!
 =================================
 
-The Rucio project is the new version of ATLAS Distributed Data Management (DDM) system services
-for allowing the ATLAS collaboration to manage the large volumes of data, both taken by the
-detector as well as generated or derived, in the ATLAS distributed computing system.
-Rucio uses to manage accounts, files, datasets and distibuted storage systems.
+Rucio is a project that provides services and associated libraries for allowing scientific
+collaborations to manage large volumes of data spread across facilities at
+multiple institutions and organisations. Rucio has been developed by
+`the ATLAS experiment <https://atlas.cern/>`_.
+
+Rucio offers advanced features, is highly scalable and modular. It is a data management
+solution that could cover the needs of different communities in the scientific
+domain (e.g., HEP, astronomy, biology).
```

Comment:
Again, here we should put the final text once checked.

Reply:
The last sentence seems a bit like advertisement. It's fine for now, but we should revise it later.